common/coin_support/eth.c

```diff
@@ -61,9 +61,63 @@
 #include "int-util.h"
 #include "logger.h"
 #include "utils.h"
+#include "abi_decode.h"
 
 static uint8_t rlp_encode_decimal(uint64_t dec, uint8_t offset, uint8_t *metadata);
 
+#define EVM_swap_TAG            (0x7c025200)
+#define EVM_swap_NUM_ARGS       10
+const Abi_Type_e EVM_swapDataType[EVM_swap_NUM_ARGS] =
+{
+    Abi_address_e,
+    Abi_address_e,
+    Abi_address_e,
+    Abi_address_e,
+    Abi_address_e,
+    Abi_uint256_e,
+    Abi_uint256_e,
+    Abi_uint256_e,
+    Abi_bytes_dynamic_e,
+    Abi_bytes_dynamic_e
+};
+
+const char *EVM_swap_Title = "Function: swap";
+const char *EVM_swap_Signature = "swap(address,(address,address,address,address,uint256,uint256,uint256),bytes,bytes)";
+
+#define EVM_uniswapV3Swap_TAG           (0xe449022e)
```

Comment:
Check once if these references are okay to add.

```suggestion
/// Ref: https://www.4byte.directory/signatures/?bytes4_signature=0xe449022e
#define EVM_uniswapV3Swap_TAG           (0xe449022e)
```

Reply:
added